src/lib/storage/entities/verifiableCredentialEntity.ts

```diff
@@ -1,25 +1,12 @@
-import {
-  PrimaryColumn,
-  Entity,
-  Column,
-  OneToMany,
-  ManyToOne,
-} from 'typeorm/browser'
-import {
-  PersonaEntity,
-  SignatureEntity,
-  CredentialEntity,
-} from 'src/lib/storage/entities'
```

Comment:
Thank you for fixing the circular dependency :+1: I was just fixing it myself now actually, it's causing a problem under certain conditions (in tests)

Reply:
Aaaactually turned out this is not enough to resolve the circular dependency.... `SignatureEntity` and `VerifiableCredentialEntity` both import each other (and probably there are more issues), and it's currently crashing my jest processes with an "out of heap memory" error

It's interesting that metro manages to get this right but it doesn't work with just ts-jest/babel

Reply:
Interesting typeorm issues:
https://github.com/typeorm/typeorm/issues/4526 
https://github.com/typeorm/typeorm/issues/2059

Reply:
Also turns out that someone already fixed it in a django-like way: https://github.com/typeorm/typeorm/pull/4195
Simply, string names of the entities!